src/qml/components/ConnectionSettings.qml

```diff
@@ -8,7 +8,37 @@ import QtQuick.Layouts 1.15
 import "../controls"
 
 ColumnLayout {
+    property bool snapshotImported: false
+    function setSnapshotImported(imported) {
+        snapshotImported = imported
+    }
+
     spacing: 4
+    Setting {
+        id: gotoSnapshot
+        Layout.fillWidth: true
+        header: qsTr("Import snapshot")
+        description: qsTr("Speed up the initial download")
```

Comment:
as you said correctly in the import snapshot dialog, it doesn't speed up the download itself.
So this might be misleading.

I think it's better to rephrase it, something like this might be more appropriate `Start using right away and do initial synchronization in the background` or maybe something short like `Do initial synchronization in the background`

Reply:
thanks for catching that... updated it